.travis.yml

```diff
@@ -1,13 +1,16 @@
 language: python
 python:
-  - "3.6"
+  - "3.6.2"
 
 os:
   - linux
 
+env:
+  - PIPENV_IGNORE_VIRTUALENVS=1
+
 install:
   - pip3 install pipenv
-  - pipenv install
+  - pipenv install --dev
```

Comment:
This marks the fact that it has to install also the packages needed for `dev` mode, for example the ones needed for unit-tests.